modules/mixed-arch-modify-machine-set.adoc

```diff
@@ -0,0 +1,72 @@
+//Module included in the following assembly
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-modify-machine-set"]
+
+= Adding the `arm64` bootimage to your machine set
+
+To deploy your heterogeneous cluster, you must add your generated `arm64` bootimage to the machine set that was generated during installation.  
+
+.Prerequisites 
+
+* Installed the OpenShift CLI (`oc`).
+* Generated a machine set during installation 
+
+.Procedure
+
+. Check that your machine set was created during installation:
++
+[source,terminal]
+----
+oc get machineset
+----
+. Add your generated `arm64` bootimage to the `resourceID` parameter, and then edit `vmSize` to the instance type used in your installation: 
++
+[source,terminal]
+----
+oc edit machineset <machineset> 
+----
+.Sample YAML machine set
++
+[source,yaml]
+----
+    providerSpec:
+        value:
+          apiVersion: azureproviderconfig.openshift.io/v1beta1
+          credentialsSecret:
+            name: azure-cloud-credentials
+            namespace: openshift-machine-api
+          image:
+            resourceID: /resourceGroups/<infrastructure_id>-rg/providers/Microsoft.Compute/images/<infrastructure_id> 
+          internalLoadBalancer: ""
+          kind: AzureMachineProviderSpec
+          location: <region> 
+          managedIdentity: <infrastructure_id>-identity 
+          metadata:
+            creationTimestamp: null
+          natRule: null
+          osDisk:
+            diskSizeGB: 128
+            managedDisk:
+              storageAccountType: Premium_LRS
+            osType: Linux
+          publicIP: false
+          resourceGroup: <infrastructure_id>-rg 
+          sshPrivateKey: ""
+          sshPublicKey: ""
+          subnet: <infrastructure_id>-<role>-subnet   
+          userDataSecret:
+            name: worker-user-data 
+          vmSize: Standard_D4s_v3 
+          vnet: <infrastructure_id>-vnet 
+          zone: "1" 
+---- 
+
+. [FIXME] Deplying the cluster step, or verification. Like an 'all set' step
```

Comment:
I'm not sure how to verify or check that machine set is ready to deploy the heterogeneous clusters

Reply:
`oc get machines -n openshift-machine-api` displays the status of the machines that are provisioned